File: test/python/nose2-to-pytest/unconvertible/UNCONV-003.expected.py
# HAMLET-TODO [UNCONVERTIBLE-SUCH-DSL]: nose2 such DSL has no direct pytest equivalent

# Original: it = such.A('calculator')

# Manual action required: Rewrite using standard pytest test functions or classes

it = such.A('calculator')

@it.should('add numbers')
def test_add(case):
    assert 1 + 1 == 2

File: test/java/junit5-to-testng/unconvertible/UNCONV-003.expected.java
import org.junit.jupiter.api.RepeatedTest;

public class RepeatedTestTest {
    // HAMLET-TODO [UNCONVERTIBLE-REPEATED-TEST]: JUnit 5 @RepeatedTest has no direct TestNG equivalent
    // Original: @RepeatedTest(5)
    // Manual action required: Use @Test(invocationCount = N) in TestNG
    @RepeatedTest(5)
    public void testRepeated() {
        assert true;
    }
}
